packages/services/f-feature-management/test/endToEnd.test.js

```diff
@@ -0,0 +1,20 @@
+import featureManagement from '../src/index';
+import expectations from './data/e2e-expectations.json';
+
+const { getBooleanValue, getIntegerValue, getStringValue } = featureManagement('je-web-core');
```

Comment:
You could probably use a dummy scope here

Reply:
It can actually be empty now as the e2e tests assume an empty scope